online-prediction/predict/main.py

```diff
@@ -0,0 +1,69 @@
+"""
+Copyright (c) 2021. Cognitive Scale Inc. All rights reserved.
+
+Licensed under CognitiveScale Example Code [License](https://cognitivescale.github.io/cortex-fabric-examples/LICENSE.md)
+"""
+import logging
+import uvicorn
+import numpy as np
+import pandas as pd
+
+# cortex
+from cortex import Cortex
+
+# fastapi
+from fastapi import FastAPI
+
+app = FastAPI()
+
+
+@app.post('/invoke')
+async def run(request: dict):
+    logging.info("Online Prediction: Invoke Request:{}".format(request))
+    columns = request['payload']["columns"]
+    instances = request['payload']["instances"]
+    df = pd.DataFrame(columns=columns, data=instances)
+    # Initialize Cortex Client
+    client = Cortex.client(api_endpoint=request["apiEndpoint"], token=request["token"], project=request["projectId"])
+
+    # Load Model from the experiment run
+    logging.info("Loading model artifacts from experiment run...")
+    model = await load_model(client, request["properties"]["experiment-name"], request["properties"]["run-id"])
```

Comment:
Do we need to load the model for every invoke? We can load it once and keep using it

Reply:
What if they update the model artifact in the experiment & invoke again?? Is there a way to know that??

Reply:
get last run if run_id is not provided, this way we can get an updated artifact. 
we don't have any way to know that artifact is updated.

Reply:
@dvasani-CS  The reason being we are using this template for MLops skill wizard flow and we always get a run-id when we create a model skill. Anyway for general purposes, I'm making it optional.